src/common/harvest/harvest-scheduler.js

```diff
@@ -89,7 +89,12 @@ export class HarvestScheduler extends SharedContext {
       const retry = submitMethod.method === submitData.xhr
       var payload = this.opts.getPayload({ retry: retry })
 
-      if (!payload) return
+      if (!payload) {
+        if (this.started) {
+          this.scheduleHarvest()
+        }
+        return
```

Comment:
Minor but can you refactor this & L127-129 into
```javascript
function scheduleNextHarvestIfActive() {
  if (this.started) {
    this.scheduleHarvest()
  }
}
```
Can insert this below `function onHarvestFinished (result)` on L132

Reply:
I think this is probably the right way to do it but it actually makes the file a couple lines longer. For the sake of time and to avoid an error, I'd like to leave it as is for now.